ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/FilterChips.tsx

```diff
@@ -42,98 +49,122 @@ function FilterChips({ defaultFilters, searchFilter, onDeleteGroup, onDelete }:
-                <ChipGroup categoryName="Image" isClosable onClick={() => onDeleteGroup('IMAGE')}>
-                    {images.map((image) => (
-                        <Chip key={image} onClick={() => onDelete('IMAGE', image)}>
-                            {image}
-                        </Chip>
-                    ))}
-                </ChipGroup>
-            </FlexItem>
-            <FlexItem className="pf-u-pt-xs">
-                <ChipGroup
-                    categoryName="Namespace"
-                    isClosable
-                    onClick={() => onDeleteGroup('NAMESPACE')}
-                >
-                    {namespaces.map((namespace) => (
-                        <Chip key={namespace} onClick={() => onDelete('NAMESPACE', namespace)}>
-                            {namespace}
-                        </Chip>
-                    ))}
-                </ChipGroup>
-            </FlexItem>
-            <FlexItem className="pf-u-pt-xs">
-                <ChipGroup
-                    categoryName="Cluster"
-                    isClosable
-                    onClick={() => onDeleteGroup('CLUSTER')}
-                >
-                    {clusters.map((cluster) => (
-                        <Chip key={cluster} onClick={() => onDelete('CLUSTER', cluster)}>
-                            {cluster}
-                        </Chip>
-                    ))}
-                </ChipGroup>
-            </FlexItem>
-            <FlexItem className="pf-u-pt-xs">
-                <ChipGroup
-                    categoryName="Severity"
-                    isClosable
-                    onClick={() => onDeleteGroup('Severity')}
-                >
-                    {severities.map((severity) => (
-                        <Chip key={severity} onClick={() => onDelete('Severity', severity)}>
-                            <FilterChip
-                                isGlobal={defaultFilters.Severity?.includes(severity)}
-                                name={severity}
-                            />
-                        </Chip>
-                    ))}
-                </ChipGroup>
-            </FlexItem>
-            <FlexItem className="pf-u-pt-xs">
-                <ChipGroup
-                    categoryName="Fixable"
-                    isClosable
-                    onClick={() => onDeleteGroup('Fixable')}
-                >
-                    {fixables.map((fixableStatus) => (
-                        <Chip
-                            key={fixableStatus}
-                            onClick={() => onDelete('Fixable', fixableStatus)}
-                        >
-                            <FilterChip
-                                isGlobal={defaultFilters.Fixable?.includes(fixableStatus)}
-                                name={fixableStatus}
-                            />
-                        </Chip>
-                    ))}
-                </ChipGroup>
-            </FlexItem>
```

Comment:
this was causing extra space to be taken up on the left side of the chips, even if the chip group was empty.